zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/slack/PostMessage.java

```diff
@@ -0,0 +1,47 @@
+package com.github.onsdigital.zebedee.util.slack;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PostMessage {
+    public enum Emoji {
+        HeavyExclamationMark,
```

Comment:
Naming convention for enum - upper case. Enums can have values e.g.
```
    public enum Emoji {
        HEAVY_EXCLAMATION_MARK(":heavy_exclamation_mark:"),

        CHART_WITH_UPWARDS_TREND(":chart_with_upwards_trend:");

        private final String icon;

        private Emoji(String icon) {
            this.icon = icon;
        }

        public String getIcon() {
            return icon;
        }
    }
```
then you can do `HeavyExclamationMark.getIcon()`